src/mmw/apps/modeling/tasks.py

```diff
@@ -63,11 +63,11 @@ def start_histogram_job(self, json_polygon):
     the polygon so that areas can be calculated from
     the results.
     """
-    polygon = json.loads(json_polygon)
+    polygon = ensure_aoi_4d(json.loads(json_polygon))
 
     return {
         'pixel_width': aoi_resolution(polygon),
-        'sjs_job_id': histogram_start([json_polygon], self.retry)
+        'sjs_job_id': histogram_start([json.dumps(polygon)], self.retry)
```

Comment:
I think this is problematic since we're starting a histogram job on a portion of a MultiPolygon.  This may be helpful, since it's entirely possible the analysis (to say nothing of the model steps) require a Polygon.  But in that case, we should be explicit that we're converting the geometry.  My suspicion is that RWD produces a multipolygon with a main body and a small (or several small) slivers due to anomalies in the source DEM.  In that case, it may be benificial to take the largest of the polygons in that scenario, and not just the first one.